ipython/ResamplingAndMonteCarloMethods/resampling_tutorial_2a.ipynb

```diff
@@ -60,7 +60,7 @@
    "id": "4b709400-3601-4ce5-9a36-538498abab45",
    "metadata": {},
    "source": [
-    "Like the mean comparison test in Efron's example, this is an example of an \"independent sample\" test of the null hypothesis that group labels (`x`, `y`) are entirely random. In fact, because `mannwhitneyu` claims to produce an exact value of the statistic, we would expect `permutation_test` to return precisely the same $p$-value (using `mannwhitneyu` only to compute the statistic)."
+    "Like the mean comparison test in Efron's example from [Tutorial 2](https://nbviewer.org/github/scipy/scipy-cookbook/blob/main/ipython/ResamplingAndMonteCarloMethods/resampling_tutorial_2.ipynb), this is an example of an \"independent sample\" test of the null hypothesis that group labels (`x`, `y`) are entirely random. In fact, because `mannwhitneyu` claims to produce an exact $p$-value, we would expect `permutation_test` to return precisely the same $p$-value (using `mannwhitneyu` only to compute the statistic)."
```

Comment:
Someone working through this notebook some time after the previous one may not remember Efron's example.

Reply:
Yup. I originally wrote it all in one document, I think.

I don't know if the reader will know the tutorials by number, though. Maybe "Permutation Tests", the title of the notebook?

Reply:
That's a good point about people not knowing the tutorials by number. Maybe just 
"the previous tutorial on [Permutation Tests](link)."?